src/App.js

```diff
@@ -1,13 +1,23 @@
+/* eslint-disable */
 import React from 'react';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
 import './App.css';
-import Calculator from './component/calculator';
+import Calculator from './component/Calculator';
+import Home from './component/Home';
+import Quote from './component/Quote';
+import NavBar from './component/Navbar';
```

Comment:
- [ ] <b>REQUIRED:</b> There is an error in your app. Kindly Fix it. 
 ![Screen Shot 2022-06-01 at 21 45 30](https://user-images.githubusercontent.com/45217364/171490676-b8ee2488-b1a2-4919-9a80-77c876b6e3a5.png)